### PR TITLE
Fix typos, remove Airplane, updates

### PR DESCRIPTION
**Pull request by fredster33**
Airplane is mainly micro-opts and often does unexpected things with DEAR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,12 @@ There will never be a guide that will give you perfect results. Each server has
...
 
 ## Server jar
-Your choice of server software can make a huge difference in performance and api possibilities. There are currently multiple viable popular server jars, but there are also a few that you should stay away from for various reasons.
+Your choice of server software can make a huge difference in performance and API possibilities. There are currently multiple viable popular server JARs, but there are also a few that you should stay away from for various reasons.
 
 Recommended top picks:
 * [Paper](https://github.com/PaperMC/Paper) - The most popular server software that aims to improve performance while fixing gameplay and mechanics inconsistencies.
 * [Tuinity](https://github.com/Spottedleaf/Tuinity) - Paper fork that includes even more high-performance patches.
 * [Purpur](https://github.com/pl3xgaming/Purpur) - Tuinity fork focused on features and the freedom of customization.
-* [Airplane](https://github.com/Technove/Airplane) - Tuinity fork that aims to further improve server performance.
```
Comment (YouHaveTrouble) [inline README.md:23]: I do not agree with removing notable forks from this list.
The point of this guide is to educate and inform people of available options, removing airplane from here would hurt guide's objectivity. Unfortunately you disallowed edits so I cannot fix this myself.

Comment (fredster33) [reply README.md:23]: @YouHaveTrouble sorry for the inconvenience of disallowing edits. If removing Airplane is not an option, would adding a note that DEAR can sometimes cause problems be viable?

Comment (YouHaveTrouble) [reply README.md:23]: I will add such note later on

Comment (fredster33) [reply README.md:23]: @YouHaveTrouble re-added, with a note, hopefully the note isn't too negative. feel free to suggest changes

**Comment by AlessioGr** [comment]
Please provide proof for your random claims first. Your not is not objective.

File: README.md [hunk 2]
```diff
@@ -13,15 +13,14 @@ There will never be a guide that will give you perfect results. Each server has
...
-
+* [Airplane](https://github.com/Technove/Airplane) - Tuinity fork that aims to further improve server performance. **Note: Airplane is mainly made out of micro-optimizations, most of which don't really give any noticeable gains at all outside of extreme circumstances. The few features that do actually change the game often do so in unexpected and unwanted ways, the most notable one being DEAR. The Airplane team and community as a whole, have also been known to be negative towards end-users.**
```
Comment (Titaniumtown) [inline README.md:23]: lmao what

**Comment by BumbleTree** [comment]
Airplanes performance is none disputable. And Dear doesn't break mobs - A SMP with unrestricted farms and builds with high mob spawn rates. Bogus Claims

**Comment by YouHaveTrouble** [comment]
I'm just gonna close it before we have a full on fork war here

**Comment by fredster33** [comment]
> I'm just gonna close it before we have a full on fork war here

It seems like this is fairly controversial, yes.

First, if you're talking about any percentages that Airplane claims, just note that it's very easy to get confused by percentages. Nobody knows where they come from or what they're based on, and it's hard to refute if you can't reproduce the exact benchmark those percentages came from. For example, If something causes 1% of lag, and you were to improve it by 30%, that improvement ends up being a 0.3% increase in performance, which is minor.

Second, most of Airplane's patches are micro-optimizations. These are mainly for performance changes that have so little gain that there's no real value in making those changes. The "benefits" (sometimes not even a 0.0001% increase) don't outweigh the possibility of plugins breaking or server issues arising for supposedly unforeseen reasons. Time should be put working on optimizing/recreating systems that are broken or poorly implemented, not looking for small changes to make that give hardly any benefit whatsoever.

Finally, to @AlessioGr, @Titaniumtown, MWHunter, etil2jz, and 2020Sanoj (who all reacted with emojis to my PR and proceeding comments): It seems like you're trying to "win" a PR by polling. [Polls do, and should not, cause the direct outcome of a decision.](https://meta.wikimedia.org/wiki/Polls_are_evil). A proper community discussion, especially in open-source areas, is a result of discussion, presentation of logical argumentation, and community consensus as a result of earlier.

This may look like I'm trying to (re)start a fork war here, but really, I'm just expressing my concerns on the current status of the forks list.

I do acknowledge that every server and user will have their unique needs, and there should not be a "one-size-fits-all" solution. *However,* I am concerned on the addition of Airplane without any note that DEAR can be immensely troublesome for the majority of use cases.

Regarding this PR, I'm totally fine with having it closed. It looks like the rest of my changes are uncontroversial, so if it's fine, I'd like to resubmit the uncontroversial ones.

I would like to credit the Purpur community for providing extremely useful reference information that I used to research and write this comment. Some information was copied verbatim; this comment meets the legal criteria of Fair Use for the purposes of critique and private research, and no citation herein constitutes infringement under the Digital Millennium Copyright Act.

**Comment by fredster33** [comment]
Before the pitchforks (presumably) come back again, I'll clarify my positioning.

Some smart people showed me some smart data on how Airplane is performant. I was wrong, and I sincerely apologize for the mini-chaos I caused here. Airplane has changed significantly since I first looked at it, and I was too stubborn.

I still stand for my comments on polls, percentages, and no one-size-fits-all solution, but I now am a fan of Airplane.

**Comment by Titaniumtown** [comment]
> Finally, to @AlessioGr, @Titaniumtown, MWHunter, etil2jz, and 2020Sanoj (who all reacted with emojis to my PR and proceeding comments): It seems like you're trying to "win" a PR by polling. Polls do, and should not, cause the direct outcome of a decision.. A proper community discussion, especially in open-source areas, is a result of discussion, presentation of logical argumentation, and community consensus as a result of earlier.

😆 This is just how PRs work, you submit one and people respond.

**Comment by fredster33** [comment]
> > Finally, to @AlessioGr, @Titaniumtown, MWHunter, etil2jz, and 2020Sanoj (who all reacted with emojis to my PR and proceeding comments): It seems like you're trying to "win" a PR by polling. Polls do, and should not, cause the direct outcome of a decision.. A proper community discussion, especially in open-source areas, is a result of discussion, presentation of logical argumentation, and community consensus as a result of earlier.
> 
> 😆 This is just how PRs work, you submit one and people respond.

For future reference, I'm referring to community theory, which often isn't the case in real practice :)

**Pull request closed without merging**